src/WelcomeArea.tsx

```diff
@@ -129,45 +130,109 @@ export const WelcomeArea = () => {
         )}
         {state.matches('welcomeArea') && (
           <>
-            <Stack spacing="6">
-              <Stack>
-                <Title>XState Visualizer</Title>
-                <Description>
-                  Welcome to the visualizer! Here, you can build state machine
-                  diagrams using XState.
-                </Description>
-              </Stack>
-              <Stack spacing="3">
-                <Button
-                  {...buttonStyleProps}
-                  onClick={() => send('CLICK_SEE_EXAMPLE')}
-                >
-                  <HStack spacing="4">
-                    <ViewIcon color="gray.200" h="6" w="6" />
-                    <Text color="gray.100">See an example</Text>
-                  </HStack>
-                </Button>
-                <Button
-                  {...buttonStyleProps}
-                  as={Link}
-                  href="http://xstate.js.org/docs"
-                  rel="noreferrer"
-                  target="_blank"
-                >
-                  <HStack spacing="4">
-                    <LightbulbIcon color="gray.200" h="6" w="6" />
-                    <Text color="gray.100">Learn how to build machines</Text>
-                  </HStack>
-                </Button>
-                <Button
-                  {...buttonStyleProps}
-                  onClick={() => send('CLICK_START_CODING')}
-                >
-                  <HStack spacing="4">
-                    <BoltIcon color="gray.200" h="6" w="6" />
-                    <Text color="gray.100">Start coding right away</Text>
-                  </HStack>
-                </Button>
+            <Stack spacing="3">
+              <Stack spacing="6">
+                <Title>
+                  <Heading size="lg">Stately Visualizer</Heading>
+                </Title>
+                <Text color="gray.400" size="md">
+                  Build state machine diagrams using XState.
+                </Text>
+                <Stack backgroundColor="gray.700" spacing="5" padding="6">
+                  <Text>
+                    This legacy visualizer is deprecated and is no longer
+                    maintained.
+                  </Text>
+                  <Text>
+                    For the best experience,{' '}
+                    <Link
+                      href="https://stately.ai?source=viz"
```

Comment:
Why not linking to the editor directly?
`https://stately.ai/editor?source=viz`

Reply:
Good catch, thank you! Fixed.